src/EventTickets/Routes/CreateEvent.php

```diff
@@ -25,10 +29,10 @@ public function registerRoute()
             $this->endpoint,
             [
                 [
-                    'methods' => 'POST',
+                    'methods' => WP_REST_Server::CREATABLE,
                     'callback' => [$this, 'handleRequest'],
                     'permission_callback' => function () {
-                        return current_user_can( 'manage_options' );
+                        return current_user_can('publish_give_payments');
```

Comment:
Since we don't have a specific permission for this dataset, does it make sense to authorize against a payment capability? Does it suffice to fallback to a simple "is admin" (`manage_options`) check?

Reply:
I think the answer in the second comment applies to this as well.